message.go

```diff
@@ -88,6 +89,17 @@ func (cli *Client) parseMessageSource(node *waBinary.Node, requireParticipant bo
 		} else {
 			source.Chat = from.ToNonAD()
 		}
+	} else if from.IsBot() {
+		source.Sender = from
+		meta := node.GetChildByTag("meta")
+		ag = meta.AttrGetter()
+		targetChatJid := ag.JID("target_chat_jid")
```

Comment:
https://go.dev/wiki/CodeReviewComments#initialisms (applies to a lot of places)

Reply:
so no camel case

Reply:
I'll review my own code using Github here to see where this really applies

Reply:
It's still camel case, but initialisms are special and must be in consistent case rather than being treated like normal words